sdk/src/feature_set.rs

```diff
@@ -573,14 +573,6 @@ pub mod disable_turbine_fanout_experiments {
     solana_sdk::declare_id!("Gz1aLrbeQ4Q6PTSafCZcGWZXz91yVRi7ASFzFEr1U4sa");
 }
 
-pub mod drop_merkle_shreds {
```

Comment:
I can't remember - should we keep these around to serve as a "tombstone"? I know the firedancer team faced some frustrations with features that once existed, but were not in this list. But, if we've been ripping out other stuff, I guess continue to do that until we have a sweeping change

Reply:
well, if the feature have never been activated, they don't exist on-chain.
so I am not aware of any problems that might cause.